simpleclient_bom/pom.xml

```diff
@@ -0,0 +1,126 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>io.prometheus</groupId>
+        <artifactId>parent</artifactId>
+        <version>0.9.1-SNAPSHOT</version>
+    </parent>
+
+    <artifactId>simpleclient_bom</artifactId>
+    <packaging>pom</packaging>
+
+    <name>Prometheus Java Simpleclient BOM</name>
+    <description>
+        Bill of Materials for the Simpleclient.
+    </description>
+
+    <licenses>
+        <license>
+            <name>The Apache Software License, Version 2.0</name>
+            <url>http://www.apache.org/licenses/LICENSE-2.0.txt</url>
+            <distribution>repo</distribution>
+        </license>
+    </licenses>
+
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>io.prometheus</groupId>
+                <artifactId>simpleclient</artifactId>
+                <version>${project.version}</version>
```

Comment:
Please don't use variables, there's some build system out there that can't handle them.

Reply:
Okay, didn't know. Which one?

I'll try to fix that. Without variables, we'll need some special handling for releases, so that versions are changed accordingly. Let's see...

Reply:
It was some years ago, so I don't recall offhand.

I'd expect that mvn would update everything automatically, as it already does that for the other versions.

Reply:
You're right. The release plugin handles this out of the box just fine. It seems to rewrite versions for all projects in the reactor everywhere. I thought it would just update the parent version.